spec.md

```diff
@@ -21,12 +21,14 @@
   - [Polling Last Operation for Service Instances](#polling-last-operation-for-service-instances)
   - [Polling Last Operation for Service Bindings](#polling-last-operation-for-service-bindings)
   - [Polling Interval and Duration](#polling-interval-and-duration)
+  - [Network Metadata](#network-metadata)
   - [Provisioning](#provisioning)
   - [Fetching a Service Instance](#fetching-a-service-instance)
   - [Updating a Service Instance](#updating-a-service-instance)
   - [Binding](#binding)
     - [Types of Binding](#types-of-binding)
   - [Fetching a Service Binding](#fetching-a-service-binding)
+  - [Adapting Service Binding Credentials](adapting-service-binding-credentials)
```

Comment:
The hash sign is missing in the anchor, this seems to make the travis verification job fail. (`#adapting-service-binding-credentials`).